demo/tasks-licenses.json

```diff
@@ -112,7 +112,7 @@
 	},
 	{
 		"type": "dataset",
-		"enabled": true,
+		"enabled": false,
```

Comment:
Is it what you Want?

Reply:
That was supposed to be a temporary measure to work without launching the background tasks. I made a new commit to enable the tasks